app/lib/account/models.dart

```diff
@@ -63,6 +63,25 @@ class OAuthUserID extends db.ExpandoModel {
   String get userId => userIdKey.id as String;
 }
 
+/// Maps the session id (from cookie) to User.id
+class SessionUserId extends db.ExpandoModel {
+  /// Same as [id].
+  String get sessionId => id as String;
+
+  @db.ModelKeyProperty(required: true)
+  db.Key userIdKey;
+
+  String get userId => userIdKey.id as String;
+
+  @db.DateTimeProperty(required: true)
+  DateTime created;
+
+  @db.DateTimeProperty(required: true)
+  DateTime expires;
+
```

Comment:
How about image and email...

Reply:
My first thinking was that we can cache those on the client side instead, but you are right, we should do the caching here.